<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimController</class>
 <widget class="QMainWindow" name="SimController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Environment</string>
     </attribute>
     <widget class="QPushButton" name="openDoor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Door</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CloseDoor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Close Door</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Speech</string>
     </attribute>
     <widget class="QPushButton" name="FakeSay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Fake Say</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FakeRec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Fake Rec</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SayText">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>481</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RecText">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>481</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>HRI</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Make Human Wave</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="HumanWave">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPepper_Sim_Controller">
    <property name="title">
     <string>Pepper Sim Controller</string>
    </property>
   </widget>
   <addaction name="menuPepper_Sim_Controller"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
